gamemodes/terrortown/gamemode/shared/lang/english.lua

```diff
@@ -1249,13 +1061,166 @@ L.door_locked = "This door is locked"
+L.label_crosshair_static_enable = "Enable static crosshair"
+L.label_crosshair_dot_enable = "Enable crosshair dot"
+L.label_crosshair_lines_enable = "Enable crosshair lines"
+L.label_crosshair_scale_enable = "Enable weapon dependant weapon scale"
+L.label_crosshair_ironsight_low_enabled = "Lower weapon when using ironsights"
+L.label_damage_indicator_enable = "Enable damage indicator"
+L.label_damage_indicator_mode = "Select damage indicator theme"
+L.label_damage_indicator_duration = "Seconds the damage indicator is visible after a hit"
+L.label_damage_indicator_maxdamage = "Damage needed for the maximum opacity"
+L.label_damage_indicator_maxalpha = "Damage Indicators maximum opacity"
+L.label_performance_halo_enable = "Draw an outline around some entities while looking at them"
+L.label_performance_spec_outline_enable = "Enable controlled objects' outlines"
+L.label_performance_ohicon_enable = "Enable overhead role icons"
+L.label_interface_tips_enable = "Show gameplay tips at the bottom of the screen while spectating"
+L.label_interface_popup = "Start of round info popup duration"
+L.label_interface_fastsw_menu = "Enable menu with fast weapon switch"
+L.label_inferface_wswitch_hide_enable = "Enable weapon switch menu auto-closing"
+L.label_inferface_scues_enable = "Play sound cue when a round begins or ends"
+L.label_gameplay_specmode = "Spectate-only mode (always stay spectator)"
+L.label_gameplay_fastsw = "Fast weapon switch"
+L.label_gameplay_hold_aim = "Enable hold to aim"
+L.label_gameplay_mute = "Mute living players when dead"
+L.label_gameplay_dtsprint_enable = "Enable double tap sprinting"
+L.label_gameplay_dtsprint_anykey = "Continue double tap sprinting until you stop moving"
+L.label_hud_default = "Default HUD"
+L.label_hud_force = "Forced HUD"
+
+L.label_bind_weaponswitch = "Pickup Weapon"
+L.label_bind_sprint = "Sprint"
+L.label_bind_voice = "Global Voice Chat"
+L.label_bind_voice_team = "Team Voice Chat"
+
+L.label_hud_basecolor = "Base Color"
+
+L.header_bindings_ttt2 = "TTT2 Bindings"
+L.header_bindings_other = "Other Bindings"
+L.header_language = "Language Settings"
+L.header_global_color = "Select Global Color"
+L.header_hud_select = "Select a HUD"
+L.header_hud_customize = "Customize the HUD"
+L.header_vskin_select = "Select and Customize the VSkin"
+L.header_targetid = "TargetID Settings"
+L.header_shop_settings = "Equipment Shop Settings"
+L.header_shop_layout = "Item List Layout"
+L.header_shop_marker = "Item Marker Settings"
+L.header_crosshair_settings = "Crosshair Settings"
+L.header_damage_indicator = "Damage Indicator Settings"
+L.header_performance_settings = "Performance Settings"
+L.header_interface_settings = "Interface Settings"
+L.header_gameplay_settings = "Gameplay Settings"
+L.header_roleselection = "Enable Role Assignment"
+L.header_hud_administration = "Select Default and Forced HUDs"
+L.header_hud_enabled = "Enable/Disable HUDs"
+
+L.button_menu_back = "back"
```

Comment:
Should be capital as all other buttons, shouldn't it?

```suggestion
L.button_menu_back = "Back"
```

Reply:
to be honest I like that it is lowercase (It is the button in the topleft corner)